src/app/components/nav-bar/nav-bar.component.scss

```diff
@@ -0,0 +1,25 @@
+* {
+    background-color: #CFE2F3;
+    margin: 0;
+    padding: 0;
+}
+
+.navBar {
+    display: flex;
```

Comment:
navbar should have 100% width(probably because of 8px body margin). Pls correct it

Reply:
I would appreciate bigger fonts for heroes hall and logout